### PR TITLE
Make license endpoint return number of nodes in existing orgs

### PR DESCRIPTION
**Pull request by jeremiahsnapp**
@stevendanna This fixes https://github.com/chef/chef-server/issues/437.

### CODE REVIEW

File: src/oc_erchef/apps/chef_db/priv/pgsql_statements.config [hunk 1]
```diff
@@ -20,7 +20,7 @@
 {ping, <<"SELECT 'pong' as ping LIMIT 1">>}.
 
 %% Query to count the number of nodes for license checks
-{count_nodes, <<"SELECT COUNT(*) as count FROM nodes">>}.
+{count_nodes, <<"SELECT COUNT(*) as count FROM nodes as n LEFT JOIN orgs as o ON n.org_id = o.id WHERE o.id IS NOT NULL;">>}.
```
Comment (marcparadise) [inline src/oc_erchef/apps/chef_db/priv/pgsql_statements.config]: Might be clearer to `SELECT COUNT(*) as count FROM nodes, orgs WHERE orgs.id = nodes.org_id`?

Comment (marcparadise) [inline src/oc_erchef/apps/chef_db/priv/pgsql_statements.config]: Alt if you wanna get all ANSI-compliant: `SELECT COUNT(*) FROM nodes INNER JOIN orgs ON orgs.id = nodes.org_id`

**Comment by jeremiahsnapp** [comment]
Thanks @marcparadise for hitting me in the head with the "obvious" stick. :smile: 

I updated the commit with your ANSI-compliant option.

**Comment by stevendanna** [comment]
:+1:

**Comment by marcparadise** [comment]
I'll merge once this passes: http://wilson.ci.chef.co/job/chef-server-12-build/1352

**Pull request merged**